es-app/src/components/TextListComponent.h

```diff
@@ -81,8 +81,8 @@ class TextListComponent : public IList<TextListData, T>
 	inline void setLineSpacing(float lineSpacing) { mLineSpacing = lineSpacing; }
 
 protected:
-	virtual void onScroll(int /*amt*/) { if(!mScrollSound.empty()) Sound::get(mScrollSound)->play(); }
```

Comment:
Compiler warnings:

In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/views/gamelist/BasicGameLis
tView.cpp:1:
In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/views/gamelist/BasicGameLis
tView.h:5:
/home/devel/temp/EmulationStation_Stable/es-app/src/components/TextListComponent.h:84:15: warning: 'o
nScroll' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]  
        virtual void onScroll(int /*amt*/) { if(!mScrollSound.empty()) Sound::get(mScrollSound)->play
(); }
                     ^